src/core/Microsoft.PowerToys.Settings.UI.UnitTests/ViewModelTests/PowerLauncherViewModelTest.cs

```diff
@@ -5,13 +5,21 @@
 using Microsoft.PowerToys.Settings.UI.Lib;
 using Microsoft.PowerToys.Settings.UI.Lib.ViewModels;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
 using Moq;
+using Microsoft.PowerToys.Settings.UI.UnitTests.Mocks;
+using Microsoft.PowerToys.Settings.UI.UnitTests.BackwardsCompatibility;
+using System.Globalization;
 
 namespace ViewModelTests
 {
     [TestClass]
     public class PowerLauncherViewModelTest
     {
+        // This should not be changed. 
+        // Changing it will causes user's to lose their local settings configs.
+        public const string OriginalModuleName = "PowerToys Run";
```

Comment:
I think for all modules other than ImageResizer we can use <Module>Settings.ModuleName to prevent redeclaration.